src/components/Dashboard/TodoList.vue

```diff
@@ -1024,4 +1038,15 @@ export default {
 .todo-item-toggle h3 {
   margin-bottom: 0.25rem;
 }
+
+.nr-info-btn {
+  padding-left:4rem;
+  padding-top: 0.2rem;
```

Comment:
Why isn't this button styled like the others? See class="info-btn".

Reply:
This button looks different. The text is also blue in color and View Details is also part of the button text. So I changed it to 'tile' instead of icon and the styling is a bit different from the icon type button